src/Cache.php

```diff
@@ -35,9 +34,9 @@ public function putData(string $provider, Collection $breweries): void
     }
 
     /**
-     * @param  Collection<Providers> $providers
-     * @param  Collection<Brewery> $breweries
-     * @throws JsonException
+     * @param Collection<Providers> $providers
+     * @param Collection<Brewery> $breweries
+     * @throws JsonException|\JsonException
```

Comment:
Why removed `use JsonException;`?

```suggestion
     * @throws JsonException
```